src/assets/scss/bs-4.3.1/_print.scss

```diff
@@ -57,7 +57,7 @@
 
     //
     // Printing Tables:
-    // http://css-discuss.incutio.com/wiki/Printing_Tables
+    // https://css-discuss.incutio.com/wiki/Printing_Tables
```

Comment:
This link doesn't work anymore, let's remove it.

Reply:
Done